coins/src/adapters/other/index.ts

```diff
@@ -73,3 +74,10 @@ export function manualInput(timestamp: number = 0) {
     manualInputAdapter("polygon", timestamp),
   ]);
 }
+export function realt(timestamp: number = 0) {
+  console.log("starting realt");
+  return Promise.all([
+    realtAdapter("ethereum", timestamp),
+    realtAdapter("gnosis", timestamp),
```

Comment:
chain name might need to be xdai instead

Reply:
fixed :)